script/gift_exchange/load_json_challenge.rb

```diff
@@ -157,5 +158,5 @@ def make_prompt(challenge_signup, info, prompt_type)
     make_prompt(challenge_signup, info, 'offer')
   end
 
-  challenge_signup.save!
+  challenge_signup.save!(validate: false)
```

Comment:
This is a test script but I'm curious why validation is skipped here.

Reply:
Speed. I created two gift exchanges with 2000 participants on my local machine, and got rather impatient waiting for the script to finish. Almost all of the changes I made in the `script` directory were intended to make that process faster.

I can change it back, if you'd like. I think skipping validation only saves ~30 seconds, so I don't have strong feelings either way.

Reply:
Let's keep it this way. Can you add a comment that we skip validation for speed?